content/chapter/03.tex

```diff
@@ -194,7 +195,123 @@ \subsubsection*{Sensortechnologie}
 \end{itemize}
 
 \subsubsection*{Messung der Unwucht}
-Hier kommen dann Bilder rein und wir erklären und diskutieren die Ergebnisse dann, oder?
+Wie bereits unter dem vorherigen Punkt beschrieben, existieren verschiedene Sensoren auf dem \ac{9-DOF}.
+Für die Messung der Unwucht wird zunächst allerdings nur der Beschleunigungssensor eingesetzt.
+Die von ihm gelieferten Werte zeigen am deutlichsten eine Verschiebung des Motorblockes, aufgrund von Vibrationen, an.
+
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=8cm]{images/chapter/03/measurement_arduino_a2.png}
+	\caption{Ablaufdiagramm für die Unwuchtmessung am \ac{A2}}
+	\label{fig:sequence_measure_a2}
+\end{figure}
+
+Für die erste Messreihenermittlung wurde das Programm von \ac{A2} auf den Ablauf programmiert, wie in Abbildung \ref{fig:sequence_measure_a2} dargestellt.
+Hierbei wartet \ac{A2} auf einen Interrupt von \ac{A1} um mit der Unwuchtmessung zu beginnen.
+Sobald ein weiterer Interrupt an \ac{A2} gesendet wird, kann davon ausgegangen werden, dass der Propeller eine vollständige Umdrehung absolviert hat und die Unwuchtmessung abgeschlossen werden kann.
+Dieser erste Test soll zeigen, ob mögliche Muster in den Messwerten einer einzelnen Propellerumdrehung zu sehen sind.
+Verwendet wird hierfür ein Propeller, dessen Unwucht auf der gleichen Seite, wie die Markierung ist.
+
+Die aufgezeichneten Messwerte der einzelnen Messungen für diesen ersten Test, sind in zwei Tabellen dargestellt, welche jeweils drei Messergebnisse enthalten. Die Tabelle \ref{tab:rounds_balance} enthält die Messergebnisse für einen Propeller ohne eigens erstellte Unwucht.
```

Comment:
Hier sind zwei Sätze in einer Zeile.
Beim zweiten Satz würd ich das Die weglassen